idea/Hublsoft.Task.Web/Startup.cs

```diff
@@ -28,14 +28,15 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            services.AddCors(options => options.AddDefaultPolicy(builder => builder.AllowAnyOrigin().AllowAnyMethod().AllowAnyHeader()));
```

Comment:
If adding CORS functionality, can we add restrictions for legitimate Origins only